src/interfaces/views.ts

```diff
@@ -73,6 +74,7 @@ export interface PostView {
   subscribed: boolean;
   saved: boolean;
   read: boolean;
+  creator_blocked: boolean;
```

Comment:
post and comments now return whether the creator is blocked.

In the *list* version, they are never shown (IE, your front page won't show posts by blocked post creators ), but in the individual post gets, it will still fetch them, but this `creator_blocked` can let you put a blocked next to their name if you want. I'm doing that in lemmy-ui.